build-go-payments.sh

```diff
@@ -48,7 +48,7 @@ solc_0_8_13_dir=$(realpath "$1")
     ./abigen --abi "${payments_out_dir}/ERC20Payments.abi" --bin "${payments_out_dir}/ERC20Payments.bin" --pkg payments --out "${payments_abigen_dir}/erc20_payments.go"
 
     echo "#### generating latest src/tradable/ElvToken.sol"
-    ./abigen --abi "${elv_token_out_dir}/ElvToken.abi" --bin "${elv_token_out_dir}/ElvToken.bin" --pkg tradable --out "${elv_token_abigen_dir}/ElvToken.go"
+    ./abigen --abi "${elv_token_out_dir}/ElvToken.abi" --bin "${elv_token_out_dir}/ElvToken.bin" --pkg token --out "${elv_token_abigen_dir}/ElvToken.go"
```

Comment:
`go` files are usually lower and snake case.

```suggestion
    ./abigen --abi "${elv_token_out_dir}/ElvToken.abi" --bin "${elv_token_out_dir}/ElvToken.bin" --pkg token --out "${elv_token_abigen_dir}/elv_token.go"
```

Reply:
oh yes made the changes.